db/migrate/20191002123040_create_vendor_api_tokens.rb

```diff
@@ -1,4 +1,4 @@
-class CreateVendorApiTokens < ActiveRecord::Migration[6.0]
+class CreateVendorAPITokens < ActiveRecord::Migration[6.0]
```

Comment:
Why is this needed?

Reply:
As in, Duncan renamed these here: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1937/files#diff-a0b8e3ce696609e3c61a95efee133caeR4

Reply:
The migrations can't run anymore because of https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1937. cc @duncanjbrown